static/sass/components/_post-content.scss

```diff
@@ -8,19 +8,20 @@
 .content-block {
   @include center('medium-page');
   @include grid;
+  margin: size('double-gutter') auto;
 
   @supports not (display: grid) {
     max-width: size('full-content');
   }
+}
 
-  > * {
-    @supports (display: grid) {
-      grid-column: col1-start / -1;
+.content-block-inner {
+  @supports (display: grid) {
+    grid-column: col1-start / -1;
```

Comment:
I've been thinking about this lately, wouldn't a browser just ignore anything like grid-column if it doesn't support it? Would it make sense to only include things that would apply and be different for both supported and non-supported grid features? Like, margin or max-width? Thoughts?